logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -323,10 +323,31 @@ function New-DevelopmentAppSettings([hashtable] $Settings = @{ }) {
         $newSettingsFiles += $newDevelopmentSettingsPath
 
         $credentialSettings = Merge-Hashtables @{ }, (Get-CredentialSettingsByProject)[$project]
-        $newUserSettingsPath = Join-Path $projectPath "appsettings.user.json"
+        $newUserSettingsPath = Join-Path $projectPath "appsettings.secret.json"
+        
         New-JsonFile $newUserSettingsPath $credentialSettings
```

Comment:
I think we can just take the `$credentialSettings` and convert them to json and not output a file at all.

Reply:
yes , I did . Thanks